src/types/index.ts

```diff
@@ -566,6 +565,14 @@ export interface IPopupConfig {
   txBase64?: Encoded.Transaction;
 }
 
+export interface TokenProps {
+  contractId?: string;
+  tokenPairs?: any; // TODO: find and declare types
+  tokenData?: any; // TODO: update types
```

Comment:
```suggestion
  tokenData?: IToken;
```

Reply:
Using this type here creates many ts issues in `TokenDetails.vue`

Reply:
OK. So please add the type in the comment.